.github/workflows/ci.yml

```
@@ -145,3 +145,33 @@ jobs:
           source pytest/venv/bin/activate
           cd pytest 
           pytest -m "not ci_excluded" -s -x
+
+  tee-launcher-tests:
+    name: "Run TEE Launcher pytests"
+    runs-on: warp-ubuntu-2204-x64-8x
+    timeout-minutes: 60
+    permissions:
+      contents: read
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.11"
+
+      - name: Setup virtualenv
+        run: |
+          python3 -m venv pytest/venv
+          source pytest/venv/bin/activate
+          cd tee_launcher 
+          pip install -r requirements.txt
+
+      - name: Run pytest
+        run: |
+          python3 -m venv pytest/venv
+          source pytest/venv/bin/activate
+          cd tee_launcher 
+          pytest test_launcher_config.py -v -s -x
```

what about `test_launcher.py`?

---

Also, just running `pytest` should be sufficient since it then discovers all test files.

When I run this locally I need to set `PYTHONPATH=.` for `test_launcher` to succeed. 

```suggestion
          PYTHONPATH=. pytest -vsx
```

---

@netrome do you run under with env activated? (`source venv/bin/activate`), because after that step and `pip install requirements.txt` it should be available in path